cluster/gce/gci/configure-helper.sh

```diff
@@ -2082,9 +2105,14 @@ function start-kube-controller-manager {
     params+=" --pv-recycler-pod-template-filepath-hostpath=$PV_RECYCLER_OVERRIDE_TEMPLATE"
   fi
   if [[ -n "${RUN_CONTROLLERS:-}" ]]; then
-    params+=" --controllers=${RUN_CONTROLLERS}"
-  fi
-
+    # Trim the `RUN_CONTROLLERS` value. This field is quoted which is
+    # incompatible with the `convert-manifest-params` format.
+    params+=" --controllers=${RUN_CONTROLLERS//\'}"
+  fi
+  params+=" --log-file=${LOG_PATH}"
+  params+=" --logtostderr=false"
+  params+=" --log-file-max-size=0"
```

Comment:
Could we add also --stderrthreshold=FATAL to avoid logging anything to stderr (it is the case before the change)?

My understanding is that this contributes to the issue -- any data wrote to stderr must be read by docker and from my experience docker reads that data quite slow.

I think this generates significant part of the issue.

Reply:
I meant adding stderrthreshold here and in other components as well.

Reply:
For the new klog logic, if logtostderr and alsotostderr are both false, stderrthreshold will not be considered (since no stderr would ever happpen).  
See [here](https://github.com/kubernetes/klog/pull/65/files#diff-fb778b051a785bbb2eea40d6a00a3a21R838)

In such case, no data will be written to std error whatever the log severity is.

Reply:
I see. Makes sense.